ddtrace/settings/integration.py

```diff
@@ -32,12 +33,17 @@ def __init__(self, global_config, *args, **kwargs):
         # Set internal properties for this `IntegrationConfig`
         # DEV: By-pass the `__setattr__` overrides from `AttrDict` to set real properties
         object.__setattr__(self, 'global_config', global_config)
+        object.__setattr__(self, 'integration_name', name)
         object.__setattr__(self, 'hooks', Hooks())
         object.__setattr__(self, 'http', HttpConfig())
 
-        # Set default keys/values
+        # Set default analytics configuration, default is disabled
         # DEV: Default to `None` which means do not set this key
-        self['event_sample_rate'] = None
+        # Inject environment variables for integration
+        analytics_enabled_env = get_env(name, 'analytics_enabled')
+        if analytics_enabled_env is not None:
+            analytics_enabled_env = asbool(analytics_enabled_env)
+        self.setdefault('analytics_enabled', analytics_enabled_env)
```

Comment:
Don't we also need `get_env(name, 'analytics_sample_rate', default=1.0)` here as well?

Reply:
I see we are handling the `None` case below, but we should support the env variable anyways.